code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -21,6 +21,25 @@
 	tastes = list("cheese" = 1)
 	foodtype = DAIRY
 
+/obj/item/reagent_containers/food/snacks/store/fetacheesewheel
+	name = "feta cheese wheel"
+	desc = "A big wheel of delcious Feta."
+	icon_state = "fetacheesewheel"
+	slice_path = /obj/item/reagent_containers/food/snacks/fetacheesslice
+	slices_num = 5
+	list_reagents = list("nutriment" = 15, "vitamin" = 5)
+	w_class = WEIGHT_CLASS_NORMAL
+	tastes = list("sweet" = 1)
+	foodtype = DAIRY
+
+/obj/item/reagent_containers/food/snacks/fetacheesslice
+	name = "feta cheese slice"
+	desc = "A wedge of delicious Feta. The cheese wheel it was cut from can't have gone far."
```

Comment:
`couldn't have gone far`*

`Can't have` is an indicative mood, stating an absolute fact about a situation.
`Couldn't have` on the contrary is a subjunctive mood; stating an opinion or an attitude towards a hypothetical or (possibly) counterfactual situation or condition.

Reply:
Also lowercase feta.